src/SSRv1/Controller.php

```diff
@@ -670,6 +671,57 @@ public static function bulkAdd(ServerRequestInterface $request, RequestHandlerIn
 		return $handler->handle($request);
 	}
 
+	public static function bulkImport(ServerRequestInterface $request, RequestHandlerInterface $handler): ResponseInterface {
+		//Handling bulk import from peracotta
+		$body = $request->getParsedBody();
+		/* @var Database $db */
+		$db = $request->getAttribute('Database');
+		//Get all Bulk Imports from BulkTable
+		$imports = $db->bulkDAO()->getBulkImports();
+		$delete = null;
+		$import = null;
+		if( $body !== null && count($body) > 0){
+			try {
+				//Delete import handler
+				if(isset($body['delete'])){
+					$db->bulkDAO()->deleteBulkImport($body['delete']);
+					return new RedirectResponse('/bulk/import',303);
+				}
+				//Import handler
+				if(isset($body['import'])){
+					//Decoding JSON and redirecting to Item or Product add page
+					$importElement = $db->bulkDAO()->getDecodedJSON(intval($body['import']));
+
+					if( $importElement['type'] == 'I'){
+						//TODO: Serve che ritorni a bulk import?
+						$parent = null;
+						$newItem = ItemBuilder::ofArray($importElement,null,$parent);
+						$request = $request->withAttribute('Template', 'newItemPage')->withAttribute('TemplateParameters',[
+							'add' => true,
+							'base' => $newItem,
+							'importedFrom' => intval($body['import'])
+						]);
+						return $handler->handle($request);
+					}else if( $importElement['type'] == 'P'){
+						$importElement = ProductBuilder::ofArray($importElement,$importElement['brand'],$importElement['model'],$importElement['variant']);
+						$request = $request->withAttribute('Template', 'newProductPage')->withAttribute('TemplateParameters',[
+							'add' => true,
+							'base' => $importElement,
+							'importedFrom' => intval($body['import'])
+						]);
+						return $handler->handle($request);
+					}
+
+				}
+			} catch( \Exception $e) {
```

Comment:
Why? Don't catch it unless you want to print a specific error message, there's a global handles that prints a generic error page already